### ISSUE TITLE
Reolink E1 Zoom

### ISSUE THREAD

**Issue by biggeeus** [berfenger/cam2mqtt#13]
I have an issue with my Reolink E1 Zoom, where it only reports the first motion once the docker container starts, any subsequent movement is not reported.  MQTT shows all of the Reolink features and I can change via MQTT infra red setting etc.
I have 4 other cameras which all report motion ok but they all have AI motion detection, they are a mixture of RLC-820A, RLC-510A,  RLC-810A.
I have tried reverting the firmware from the July version to the February version.  
Also motion from the E1 Zoom is reported ok into Home Assistant using its integration.

Any help would be appreciated.
Thanks 

Done some further checking and my RLC-410W also exhibits the same issue.

**Comment by berfenger** [comment]
Hi, maybe there is a problem with motion debouncing.

Please try the latest edge release using the image `acasal/cam2mqtt:edge` and set `motion_debounce: false`.
```yaml
...
      onvif:
        port: 8000
        monitor_events: true
        prefer_webhook_subscription: false
        motion_debounce: false
```
Thank you for your report.

**Comment by biggeeus** [comment]
Thanks for looking at this for me, I tried the latest edge release with motion_debounce set to false.
The good news is that I now get motion events for the Reolinklink E1 Zoom and the RLC-410W
The bad news is I don't get a motion off event when the motion has stopped.
Also I get a stream of motion events 0.2 seconds apart when the motion is happening as opposed to just one event with my other cameras.

Thanks

**Comment by berfenger** [comment]
OK. I have supposed that the camera was messing around with the motion events.

This is fixable by software adding a new optional functionality. I hope to have it implemented today or tomorrow. Keep tuned.

**Comment by berfenger** [comment]
It's now ready to test. Pull again the `edge` image and try this setting:
```yaml
      onvif:
        port: 8000
        monitor_events: true
        prefer_webhook_subscription: false
        force_motion_debounce_time: 2 seconds
```
`force_motion_debounce_time` should be larger than the time between `on` motion events from the camera. For this test, please remove the option `motion_debounce`, or change it to `true`.

**Comment by biggeeus** [comment]
Thanks for the quick turn around, the modifications made look good I am now getting on and off events when motion is happening.
On thing is happening though is that I sometimes get a suprious motion event just after motion has stopped.
For example if I set the force_motion_debounce_time to 2 seconds I get ones similar to this
18/01/2023 20:53:03(-68.25 seconds) 
off
18/01/2023 20:53:01(-2.02 seconds) 
on
and  if I set the force_motion_debounce_time to 3 seconds I get ones similar to this
18/01/2023 21:11:25(-80.73 seconds) 
off
18/01/2023 21:11:22(-3.02 seconds) 
on

this doesn't happen consistantly after every motion event perhaps only 15-20% of the motion events.

Thanks
